src/main/java/com/exactpro/th2/common/message/MessageBuilder.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.common.message;
+
+public interface MessageBuilder<Builder extends MessageBuilder<Builder, MB, R>, MB extends MetadataBuilder<MB>, R> {
+    Builder setParentEventId(String id, String bookName);
```

Comment:
I am not sure but maybe we should use EventID instead of these two parameters

Reply:
As I know we don't want to have dependencies to grpc EventID.